src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -506,6 +491,61 @@ private static StringBuilder fakeBody(final Request... dispositions)
                 .append(String.format("--%s--", BOUNDARY));
             return builder;
         }
+
+        /**
+         * A private inner class for the specific implementation
+         * for the RqMultipart. see the over ridden methods for details.
+         *
+         * @todo #517:30min re-design the class.
+         * Why the super class is encapsulated inside the class?
+         * Find an implementation without doing aggregation
+         * and extending the same class
+         */
+        private static class RqMultipartRequest implements Request {
+
+            /**
+             * Request variable encapsulated.
```

Comment:
@sebing please remove `variable encapsulated` - this doesn't bring any value. Please focus on what kind of request it is